src/common/utilities/CustomFormatting.ts

```diff
@@ -0,0 +1,109 @@
+                case "number":
+                    return { type: "number", value: node };
+                case "boolean":
+                    return { type: "booelan", value: node ? 1 : 0 };
+            }
+        }
+        const operator = node.operator;
+        const operands = node.operands.map(o => this.convertCustomFormatExpressionNodes(o));
+        return { type: "operator", value: operator, operands };
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    private evaluateCustomFormatContent = (content: ICustomFormattingExpressionNode | ICustomFormattingNode | string | number | boolean, context: any): string | number | boolean => {
+        if ((typeof content === "string" && content.charAt(0) !== "=") || typeof content === "number") return content;
+        if (typeof content === "string" && content.charAt(0) === "=") {
+            const result = this._formulaEvaluator.evaluate(content.substring(1), context);
+            return result;
+        }
+        if (typeof content === "object" && (Object.prototype.hasOwnProperty.call(content, "elmType"))) {
+            return this.renderCustomFormatContent(content as ICustomFormattingNode, context);
+        } else if (typeof content === "object" && (Object.prototype.hasOwnProperty.call(content, "operator"))) {
+            const astNode = this.convertCustomFormatExpressionNodes(content as ICustomFormattingExpressionNode);
+            const result = this._formulaEvaluator.evaluateASTNode(astNode, context);
+            if (typeof result === "object" && Object.prototype.hasOwnProperty.call(result, "elmType")) {
+                return this.renderCustomFormatContent(result, context);
+            }
+            return result;
+        }
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    public renderCustomFormatContent = (node: ICustomFormattingNode, context: any, rootEl: boolean = false): any => {
+        try {
+            if (typeof node === "string" || typeof node === "number") return node;
+            // txtContent
+            let textContent: JSX.Element | string | undefined;
+            if (node.txtContent) {
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                textContent = this.evaluateCustomFormatContent(node.txtContent, context) as any;
+            }
+            // style 
+            const styleProperties = {} as React.CSSProperties;
+            if (node.style) {
+                for (const styleAttribute in node.style) {
+                    if (node.style[styleAttribute]) {
+                        styleProperties[styleAttribute] = this.evaluateCustomFormatContent(node.style[styleAttribute], context) as string;
+                    }
+                }
+            }
+            // attributes
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            const attributes = {} as any;
```

Comment:
Seems like attributes should be of type `Record<string, string>`

Reply:
Made this change 👍